pkgs/top-level/all-packages.nix

```diff
@@ -4165,10 +4165,6 @@ with pkgs;
   nodejs-slim = nodejs-slim_22;
   corepack = hiPrio corepack_22;
 
-  nodejs_18 = throw "Node.js 18.x has reached End-Of-Life and has been removed";
-  nodejs-slim_18 = throw "Node.js 18.x has reached End-Of-Life and has been removed";
-  corepack_18 = throw "Node.js 18.x has reached End-Of-Life and has been removed";
```

Comment:
In general we want to keep aliases for removed packages for at least 1 (2?) stable releases. So instead of removing you should move those aliases to the `aliases.nix` file.

Reply:
Considering how close we are to a stable release, I would leave this at least until 25.11 to be honest.

But to be clear, it is not like those rules are well defined or enforced anyway.